droid-hal-device.inc

```diff
@@ -385,6 +389,25 @@ if (grep -q '^TARGET_ARCH := arm64' %{android_root}/device/*/*/BoardConfig*.mk);
 fi
 %endif
 
+%if 0%{?with_system_keys:1}
+KERNEL_DIRS=$(find %android_root -maxdepth 1 -type d \( -name "kernel-*.*" -o -name "kernel" \) &&
+              find %android_root -maxdepth 2 -type d -path "*/linux/kernel")
+
+# Copy trusted keys into kernel source directory
+echo "$KERNEL_DIRS" |
+while IFS= read -r kernel; do
```

Comment:
You can directly pass the find output to read, see: https://github.com/koalaman/shellcheck/wiki/SC2044

Reply:
Passing output directly is a bash-specific thing, it won't work with plain POSIX shell. Is it okay to use bashisms here?